docs/source/install/upgrades.rst

```diff
@@ -28,10 +28,7 @@ Upgrade Procedure
 
    sudo st2ctl stop
 
-2. Run the migration script (if any). See section below for |st2|
```

Comment:
Why running migration script is no longer needed/listed? I didn't try this so I don't know, but @Kami knows, so pls check with him and once he's ok I am ok.